student.rb

```diff
@@ -1,12 +1,19 @@
 require './person'
+require './class_room'
 
 class Student < Person
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
   end
 
+  attr_reader :classroom
+
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  def classroom=(classroom)
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
- [ ] Besides updating the students' array in the classroom object we also need to set the classroom of the student by assigning the classroom to `@classroom`

  ```
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```